### ISSUE TITLE
Intro and abstract can be editorialized for clarity

### ISSUE THREAD

**Issue by LPardue** [httpwg/http-extensions#1703]
There's nothing wrong with them per-se but in the space of 2 paragraphs the doc says 

* This document describes the mechanism for HTTP/3.
* Appendix A.3 of [HTTP3] describes the required updates for HTTP/2 settings to be used with HTTP/3.

So is it a new mechanism, a setting update, a new setting, or what?

**Comment by RyanTheOptimist** [comment]
Good point! That is a bit awkward. Check out #1704 and let me know what you think?
